README.md

```diff
@@ -2,62 +2,74 @@
 
 **Desktop Application for [delta.chat](https://delta.chat)**
 
-[![Build Status](https://travis-ci.org/deltachat/deltachat-desktop.svg?branch=master)](https://travis-ci.org/deltachat/deltachat-desktop)
-[![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
-
 <center><img src="README_ASSETS/screenshot.png"/></center>
 
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+[![npm test](https://github.com/deltachat/deltachat-desktop/workflows/npm%20test/badge.svg)](https://github.com/deltachat/deltachat-desktop/actions?query=workflow%3A%22npm+test%22+branch%3Amaster)
+[![Build Status](https://travis-ci.org/deltachat/deltachat-desktop.svg?branch=master)](https://travis-ci.org/deltachat/deltachat-desktop)
+
 **If you are upgrading:** please see [`UPGRADING.md`](UPGRADING.md).
 
 ## Table of Contents
 
 <details><summary>Click to expand</summary>
 
 - [Install](#install)
-- [Configuration and Databases](#configuration-and-databases)
-- [Troubleshooting](#troubleshooting)
+  - [Linux](#linux)
+    - [Flatpak](#flatpak)
+    - [Arch Linux](#arch-linux)
+  - [Mac OS](#mac)
+  - [From Source](#source)
+  - [Troubleshooting](#troubleshooting)
+- [Configuration and Databases](#config-and-db)
 - [How to Contribute](#how-to-contribute)
+- [Logging](#logging)
 - [License](#license)
 
 </details>
 
-## Install
+## Install <a id="install"></a>
 
-The application can be downloaded from the [`Releases`](https://github.com/deltachat/deltachat-desktop/releases) page. Here you'll find prebuilt releases for all supported platforms. See below for platform specific instructions. If you run into any problems please consult the [Troubleshooting](#troubleshooting) section below.
+The application can be downloaded from **<https://get.delta.chat>**. Here you'll find prebuilt releases for all supported platforms. See below for platform specific instructions. If you run into any problems please consult the [Troubleshooting](#troubleshooting) section below.
```

Comment:
Let's not use prebuilt as a word for a binary release. This might be confusing for the real prebuilts, the binaries that are bundled inside the npm package of deltachat-node.